CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 * <How do the end-user experience this issue? what was the impact?> ([#????](https://github.com/realm/realm-core/issues/????), since v?.?.?)
 * Fix exception when decoding interned strings in realm-apply-to-state tool. ([#5628](https://github.com/realm/realm-core/pull/5628))
 * Fix some warnings when building with Xcode 14 ([PR #5577](https://github.com/realm/realm-core/pull/5577)).
+* Throw `runtime_error` if subscription set is requested and FLX is not enabled. ([#5637](https://github.com/realm/realm-core/pull/5637))
```

Comment:
General question: Is this how our users know flexible sync as? I think this will go into the release notes and it may confuse some users.